docs/equinix/index.md

```diff
@@ -1,2 +1,196 @@
+
+```bash
+curl -sfL https://get.rke2.io | INSTALL_RKE2_CHANNEL=v1.21 sh - ;
+systemctl enable rke2-server.service;
+systemctl start rke2-server.service;
+```
+
+* Optionally for fourth+ servers enable/start `rke2-agent.service` instead of `rke2-server.service`
+
+```bash
+systemctl enable rke2-agent.service
+systemctl start rke2-agent.service
+```
+
+## [Install Longhorn](https://longhorn.io/docs/1.2.0/deploy/install/install-with-helm/)
+
+* Add Helm chart repo
+
+```bash
+helm repo add longhorn https://charts.longhorn.io
+```
+
+* Install longhorn
+
+```bash
+kubectl create namespace longhorn-system
+helm -n longhorn-system install longhorn longhorn/longhorn
+```
+
+## [Install MetalLB](https://metallb.universe.tf)
+
+* Enable BPG for all systems in a cluster via Equinix Metal console
+* Request external IP range in the same region via Equinix Metal console (we
+  chose /30 for each cluster)
+* Gather hostnames and internal (`bond0:0`) addresses
+* Create YAML files for the cluster:
+
+```yaml
+controller:
+  image:
+    tag: main
+speaker:
+  image:
+    tag: main
+
+configInline:
+  peers:
+    - peer-address: 169.254.255.1
+      source-address: <system 1 bond0:0 IP address>
+      router-id: <system 1 bond0:0 IP address>
+      peer-asn: 65530
+      my-asn: 65000
+      password: <"Project BGP password">
+      node-selectors:
+        - match-labels:
+            kubernetes.io/hostname: <system 1 hostname>
+    - peer-address: 169.254.255.2
+      source-address: <system 1 bond0:0 IP address>
+      router-id: <system 1 bond0:0 IP address>
+      peer-asn: 65530
+      my-asn: 65000
+      password: <"Project BGP password">
+      node-selectors:
+        - match-labels:
+            kubernetes.io/hostname: <system 1 hostname>
+
+    - peer-address: 169.254.255.1
+      source-address: <system 2 bond0:0 IP address>
+      router-id: <system 2 bond0:0 IP address>
+      peer-asn: 65530
+      my-asn: 65000
+      password: <"Project BGP password">
+      node-selectors:
+        - match-labels:
+            kubernetes.io/hostname: <system 2 hostname>
+    - peer-address: 169.254.255.2
+      source-address: <system 2 bond0:0 IP address>
+      router-id: <system 2 bond0:0 IP address>
+      peer-asn: 65530
+      my-asn: 65000
+      password: <"Project BGP password">
+      node-selectors:
+        - match-labels:
+            kubernetes.io/hostname: <system 2 hostname>
+
+  address-pools:
+    - name: default
+      auto-assign: true
+      protocol: bgp
+      addresses:
+      - <external IP address range requested/30>
+```
+
+* Add Helm chart repo
+
+```bash
+helm repo add metallb https://metallb.github.io/metallb
+```
+
+* Install MetalLB
+
+```bash
+kubectl create namespace metallb-system
+helm -n metallb-system install metallb metallb/metallb \
+  -f <config file from step 2.yaml>
+```
+
+## Setting up Sextant
+
+[Link to Sextant Install goes here](https://docs.blockchaintp.com/en/latest/sextant/daml/overview/)
```

Comment:
As discussed let's do the instructions inline here. Start with e.g.

Having created a Kubernetes cluster we can now install Sextant ...